examples/platforms/simulation/ble.c

```diff
@@ -26,50 +26,204 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include "platform-simulation.h"
+
+#include <errno.h>
+
+#include <stdio.h>
+#include <stdlib.h>
 #include <openthread/platform/ble.h>
 
+#include "common/code_utils.hpp"
+#include "openthread/error.h"
+#include "utils/code_utils.h"
+
+#define PLAT_BLE_MSG_DATA_MAX 512
+uint8_t otPlatBleMsgBuf[PLAT_BLE_MSG_DATA_MAX];
```

Comment:
Change the name from `otPlatBleMsgBuf` to `sBleBuffer`? (I guess we can also define it as `static`?)

`ot` prefix is for OT defined APIs/types, so should not use it here?

Reply:
fixed.